backend/routes/db.py

```diff
@@ -725,13 +726,15 @@ def next_api_call_group_id() -> int:
     return id
 
 
-def usage_query():
+def usage_query(verbose=True):
     """Query for usage data"""
+    t0 = datetime.now()
     with get_db_connection() as con:
         data = sql_query(con, """
             SELECT DISTINCT r.*, array_sort(g.api_calls) api_calls, g.duration_seconds, g.group_start_time
             FROM public.apiruns_grouped g
             RIGHT JOIN public.api_runs r ON g.api_call_group_id = r.api_call_group_id""")
+    print(f'usage_query(): Fetched {len(data)} records in n seconds: {(datetime.now() - t0).seconds}')
```

Comment:
## `usage_query()` refactor
I added this function to `develop` already. Here, I added a printout of how long it takes. I haven't run since I added this line, but earlier when we were on call it felt like it completed in 3 minutes or so.